recipe/patches/0001-Redis-deps-now-build-but-do-not-link.patch

```diff
@@ -48,78 +49,125 @@ index f488d607f..234d72e68 100644
 +    ],
      tags = ["local"],
  )
- 
 diff --git a/bazel/ray_deps_setup.bzl b/bazel/ray_deps_setup.bzl
-index b5c1beb54..f2b372fcc 100644
+index fd844d66d..be72f6cbf 100644
 --- a/bazel/ray_deps_setup.bzl
 +++ b/bazel/ray_deps_setup.bzl
-@@ -102,6 +102,7 @@ def ray_deps_setup():
+@@ -102,7 +102,7 @@ def ray_deps_setup():
+         url = "https://github.com/redis/redis/archive/refs/tags/7.0.5.tar.gz",
          sha256 = "40827fcaf188456ad9b3be8e27a4f403c43672b6bb6201192dc15756af6f1eae",
          patches = [
-             "@com_github_ray_project_ray//thirdparty/patches:redis-quiet.patch",
-+            "@com_github_ray_project_ray//thirdparty/patches:redis-deps-ar.patch",
+-            "@com_github_ray_project_ray//thirdparty/patches:redis-quiet.patch",
++            "@com_github_ray_project_ray//thirdparty/patches:0001-keep-redis-quiet-and-override-AR.patch",
          ],
+         workspace_file_content = 'workspace(name = "com_github_antirez_redis")'
      )
- 
-diff --git a/thirdparty/patches/redis-deps-ar.patch b/thirdparty/patches/redis-deps-ar.patch
-new file mode 100644
-index 000000000..f778e26f4
---- /dev/null
-+++ b/thirdparty/patches/redis-deps-ar.patch
-@@ -0,0 +1,75 @@
-+From 3b634ca40c7876d58d571a6d99780558c791d5e3 Mon Sep 17 00:00:00 2001
-+From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-+Date: Thu, 11 Mar 2021 13:09:26 +0300
-+Subject: [PATCH] Enable overriding AR and RANLIB
+diff --git a/thirdparty/patches/redis-quiet.patch b/thirdparty/patches/0001-keep-redis-quiet-and-override-AR.patch
+similarity index 53%
+rename from thirdparty/patches/redis-quiet.patch
+rename to thirdparty/patches/0001-keep-redis-quiet-and-override-AR.patch
+index d03438759..27c01310e 100644
+--- a/thirdparty/patches/redis-quiet.patch
++++ b/thirdparty/patches/0001-keep-redis-quiet-and-override-AR.patch
+@@ -1,14 +1,45 @@
++From 10feb7a20122e021efb27d3eebcf55424f3c8d55 Mon Sep 17 00:00:00 2001
++From: mehrdadn <mehrdadn@users.noreply.github.com>
++Date: Tue, 5 May 2020 10:47:49 -0700
++Subject: [PATCH] keep redis quiet and override AR
++
++Patch to keep redis quiet maintained by upstream ray team at
++https://github.com/ray-project/ray/blob/master/thirdparty/patches/redis-quiet.patch
```

Comment:
@richardliaw or @krfricke I wonder if you guys can take this patch-to-Ray-patch to your Ray upstream, it's really dead simple - just allow to override default `ar` and its flags with properly set environment variables.

Reply:
See ray-project/ray#29323 which is waiting for replies